examples/terraform/digitalocean/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "cluster_name" {
+  description = "Name of cluster"
+  type        = string
+  default     = "talos-do"
+}
+
+variable "num_control_plane" {
+  description = "Number of control plane nodes to create"
+  type        = number
+  default     = 1
```

Comment:
The README states that 3 control plane nodes will be created, but the default specified here is 1.

Reply:
good call, thank you! fixed in amended commit.